cactusbot/services/beam/handler.py

```diff
@@ -79,17 +77,23 @@ def __init__(self, channel):
     async def handle_liveloading(self, packet):
         """Handle liveloading packets."""
 
+        packet = json.loads(packet)
+
         data = packet.get("data")
-        if not isinstance(data, list) or not isinstance(data[0], str):
+        if not isinstance(data, dict):
+            return
+
+        event = data["channel"].split(":")[-1:][0]
+        data = data.get("payload")
```

Comment:
We're already using `data`... should probably either not define a new variable for `payload`, or give it a different name.